src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -698,7 +698,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     needed in source build.
   -->
   <Target Name="_InstallFrameworkIntoLocalDotNet"
-          Condition="'$(DotNetBuildFromSource)' != 'true'"
+          Condition="'$(DotNetBuild)' != 'true'"
```

Comment:
If we do need some kind of local dotnet for testing (not building!), it should be not the ci dotnet. It should be something under artifacts.